tldr/semantic.py

```
@@ -733,6 +733,114 @@ def _process_file_for_extraction(
+                    params = []
+                    for child in node.children:
+                        if child.type == "formal_parameters":
+                            for param in child.children:
+                                if param.type == "simple_parameter" or param.type == "variadic_parameter":
+                                    for param_child in param.children:
+                                        if param_child.type == "variable_name":
+                                            params.append("$" + content[param_child.start_byte:param_child.end_byte])
+                                            break
+                    return f"function {name}({', '.join(params)})"
+
+                def walk_tree(node):
```

Ruff (0.14.13)

Missing return type annotation for private function `walk_tree`

Add return type annotation: `None`

(ANN202)